waspc/cli/src/Wasp/Cli/Parser.hs

```diff
@@ -0,0 +1,64 @@
+module Wasp.Cli.Parser (parserRunnerSettings) where
+
+import Options.Applicative (Parser, ParserInfo, ParserPrefs, (<**>), (<|>))
+import qualified Options.Applicative as O
+import Options.Applicative.Help (text)
+import Wasp.Cli.Command.Call (Call (..))
+import Wasp.Cli.Command.CreateNewProject.Parser (new)
+import Wasp.Cli.Command.Db.Parser (db)
+import Wasp.Cli.Command.Deploy.Parser (deploy)
+import Wasp.Cli.Command.ShellCompletion.Parser (completion)
+import Wasp.Cli.Command.Start.Parser (start)
+import Wasp.Cli.Command.Test.Parser (test)
+import Wasp.Cli.Command.WaspLS.Parser (waspls)
+import Wasp.Cli.Parser.Util (mkCommand)
+import qualified Wasp.Util.Terminal as Term
+
+parserRunnerSettings :: (ParserPrefs, ParserInfo Call)
+parserRunnerSettings = (preferences, options)
+  where
+    preferences = O.prefs O.showHelpOnEmpty
+    footer =
+      text $
+        unlines
+          [ Term.applyStyles [Term.Green] "Docs:" <> " https://wasp-lang.dev/docs",
+            Term.applyStyles [Term.Magenta] "Discord (chat):" <> " https://discord.gg/rzdnErX",
+            Term.applyStyles [Term.Cyan] "Newsletter:" <> " https://wasp-lang.dev/#signup"
+          ]
+    options =
+      O.info (parserSuite <**> O.helper) $ O.footerDoc (Just footer)
+
+parserSuite :: Parser Call
```

Comment:
Oh! I take inspiration from `testSuite` commonly used to group bunch of smaller tests together. We could perhaps name it something like `parseGroup` or similar, maybe?

Reply:
:thinking: Hmm. alright, I'll try. 

>This function handles three set of subparsers. Each of these subparsers are in charge of parsing all top level commands. Based on first CLI arg (new, compile, db, etc), it will call either subparser to handle the rest of CLI args.

With this, I think it could be `topLevelCommandsParser`, perhaps? 

Also, after reading your thoughts about the suffix, I decided to rename the parsers with `Parser` suffix. Personally, I think this makes more sense because it's more consistent with the parser return type `Parser a`.

Pushed these changes to 52bbc514343b65888ffb87a941d827bc9913a813.

Reply:
Both sounds good, nice!